src/boot/efi/meson.build

```diff
@@ -102,6 +102,46 @@ if have_gnu_efi
         efi_conf.set10('ENABLE_TPM', get_option('tpm'))
         efi_conf.set('SD_TPM_PCR', get_option('tpm-pcrindex'))
 
+        if get_option('sbat_distro') != ''
+                efi_conf.set_quoted('SBAT_PROJECT', meson.project_name())
+                efi_conf.set_quoted('PROJECT_VERSION', substs.get('PROJECT_VERSION'))
+                efi_conf.set_quoted('PROJECT_URL', substs.get('PROJECT_URL'))
+                efi_conf.set_quoted('SBAT_DISTRO_GENERATION', '@0@'.format(get_option('sbat_distro_generation')))
+                sbatvars = [['sbat_distro', 'ID'],
+                            ['sbat_distro_summary', 'NAME'],
+                            ['sbat_distro_url', 'BUG_REPORT_URL']]
+                foreach sbatvar : sbatvars
+                        value = get_option(sbatvar[0])
+                        if value == '' or value == 'auto'
+                                value = run_command('sh', '-c', '. /etc/os-release && echo $' + sbatvar[1]).stdout().strip()
```

Comment:
The official interface is that `/etc/os-release` should be loaded first, and if not found, `/usr/lib/os-release`. Please do the same here.

Reply:
See `test/units/testsuite-50.sh` for os-release handling.

Reply:
but also `/etc/os-release` is usually a symlink to /usr/lib/os-release as setup by our own tmpfiles. Or are there any builders that don't have that?

also somehow

```
bash -c '. /etc/os-release-nonexistant 2>/dev/null || . /usr/lib/os-release; echo $ID'
ubuntu
```
works in bash, but not in dash =(

```
$ dash -c '. /etc/os-release-nonexistant 2>/dev/null || . /usr/lib/os-release; echo $ID'
$
```

i love shell

Reply:
We don't care about dash too much, and we require bash for various things anyway. So just make sure to call `bash` not `sh` in the command.